test/passing/tests/doc_repl.mld.err

```diff
@@ -0,0 +1,22 @@
+Warning: Invalid documentation comment:
+File "tests/doc_repl.mld", line 28, character 2 to line 31, character 0:
+invalid code block: not expecting: unexpected character 'v'.
+Hint: did you forget a space after the '#' at the start of the line?
```

Comment:
These warnings should be silenced. Code blocks can contain many things that don't parse, for example here, MDX toplevel syntax. They spam the output and users might not want to fix them.

This can show unexpected output too, for example the `Invalid literal 2ceaf76b9f84cb632327c1479d0f30acfa3eeba2` error in my comment above, which is printed with "error" in red.

My suggestion: Remove warnings when the language of the block is not specified. (of course keep parsing)

Reply:
Done in #2151